src/Storage.js

```diff
@@ -34,6 +36,18 @@ function extractSettings({
   };
 }
 
+function mapJoiError(error) {
+  return `\n${_.chain(error.details).map(d => `- ${d.message}`).join('\n').value()}`;
+}
+
+function validateId(id) {
+  const { error } = Joi.validate(id, Joi.string().length(16).hex().required());
```

Comment:
This validation is not being done by storage microservice?
I don't think that is good this validation be in the SDK.

Reply:
+1

If you to validate to avoid the user to pass an empty value, I suggest to only verify the it is non-empty (or that is a string).

Reply:
Done.